packages/components/inputs/rich-text-utils/src/plugins/redo.ts

```diff
@@ -1,42 +1,47 @@
+import { KeyboardEvent } from 'react';
 import { isKeyHotkey } from 'is-hotkey';
 import memoize from 'lodash/memoize';
+import type { TEditor, TOptions } from '../editor.types';
 
 const memoizedIsHotkey = memoize(isKeyHotkey);
 
 const HOT_KEY = 'mod+y';
 
-const hasRedos = (editor) => {
+const hasRedos = (editor: TEditor) => {
   const { value } = editor;
   const { data } = value;
   const redos = data.get('redos');
   return redos && redos.size > 0;
 };
 
-const RedoPlugin = (options = {}) => {
+const RedoPlugin = (options = {} as TOptions) => {
   const hotkey = options.hotkey || HOT_KEY;
   const isRedoHotkey = memoizedIsHotkey(hotkey);
 
   return [
     {
-      // eslint-disable-next-line consistent-return
-      onKeyDown(event, editor, next) {
+      onKeyDown(
+        event: KeyboardEvent<HTMLInputElement | HTMLButtonElement>,
+        editor: TEditor,
+        next: () => void
+      ) {
         if (!isRedoHotkey(event) || !hasRedos(editor)) {
           return next();
         }
 
         event.preventDefault();
-        editor.toggleRedo();
+        editor.toggleRedo!();
```

Comment:
Is this correct? Or did you mean this?

```suggestion
        editor.toggleRedo?.();
```